examples/python/README.md

```diff
@@ -2,7 +2,7 @@
 
 In this section you will find the complete examples from the [Wallaroo Python](/book/python/intro.md) section, as well as some additional examples. Each example includes a README describing its purpose and how to set it up and run it.
 
-To get set up for running examples, if you haven't already, refer to [Building a Python application](/book/python/building.md).
+To get set up for running examples, if you haven't already, refer to [Choosing an Installation Option for Wallaroo](/book/getting-started/choosing-an-installation-option.md).
```

Comment:
This should point to docs.wallaroolabs.com (see #1956)

Reply:
Ah, yes. Ok, fixed.